app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user, foreign_key: 'user_id'
```

Comment:
- **[OPTIONAL]** You don't really need to specify the foreign key for the `user` association here. Rails knows you are referring to the `User` model, so it looks for a `user_id` by itself. If the foreign key was different from the model name then it would make sense to specify foreign key name.